ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -72,6 +72,19 @@ def BitwiseOr(*, x, y, name="BitwiseOr"):
     return ivy.bitwise_or(x, y)
 
 
+@to_ivy_arrays_and_back
+def avg_pool_3d(x, ksize, strides, padding, name=None):
+    with ivy.name_scope(name or 'avg_pool_3d'):
+        output = ivy.raw_ops.AvgPool3D(
```

Comment:
You should ivy.avg_pool3d(), this function https://github.com/unifyai/ivy/blob/master/ivy/functional/ivy/experimental/layers.py#L366, as our backend is in ivy and frontend in the native framework.